src/DynamoCoreWpf/Views/Search/LibrarySearchView.xaml

```diff
@@ -654,10 +660,15 @@
                                         AllowsTransparency="True"
                                         MouseLeave="OnPopupMouseLeave">
             <uicontrols:LibraryToolTipPopup.IsOpen>
-                <Binding ElementName="libraryToolTipPopup"
-                         Path="DataContext"
-                         Mode="OneWay"
-                         Converter="{StaticResource NodeSearchElementVMToBoolConverter}" />
+                <MultiBinding Converter="{StaticResource MultiBoolToBoolConverter}">
+                    <Binding Path="DataContext.ShowTooltipForTopResult"
+                             Mode="OneWay"
+                             RelativeSource="{RelativeSource FindAncestor, AncestorType={x:Type search:SearchView}}"/>
+                    <Binding ElementName="libraryToolTipPopup"
+                             Path="DataContext"
+                             Mode="OneWay"
+                             Converter="{StaticResource NodeSearchElementVMToBoolConverter}" />
+                </MultiBinding>
```

Comment:
Okay here's the thing: closing tool-tip with delay requires a timer, and when that timer fires, the bound property is updated and then the tool-tip closes. Right? So I'm wondering if the timer fires, can we not just do `libraryToolTipPopup.IsOpen = false` so that it closes (after a while)?